src/plugins/score-plugin-avnd/Spat/Rotator.hpp

```diff
@@ -0,0 +1,152 @@
+#pragma once
+#include <iostream>
+//#include <vector>
+//#include <iomanip>
+#include <halp/audio.hpp>
+#include <halp/controls.hpp>
+#include <halp/meta.hpp>
+//#include <experimental/mdspan>
+
+#include <saf.h>
+//#include <rotator.h>
+//#include <cblas.h>
+
+#define MAX_ORDER 7
```

Comment:
In modern C++ we use constexpr instead of defines: 

     class Rotator { 
     static constexpr int max_order = 7;
    };